src/org/labkey/test/tests/PivotQueryTest.java

```diff
@@ -201,18 +201,16 @@ public void testBadPivotQuery()
         datasetDesigner.getFieldsPanel().addFields(List.of(textField));
         var viewDatasetPage = datasetDesigner.clickSave()
                 .clickViewData();
-        TestDataUtils.TsvQuoter _tsvQuoter = new TestDataUtils.TsvQuoter();
-        String bulkData = """
-                ParticipantId	date	[F1]
-                1	7/28/2025	this
-                2	7/29/2025	that
-                3	7/30/2025	the other
-                4	7/31/2025	and more
-                5	8/1/2025	but wait
-                6	8/2/2025	still more
-                """.replace("[F1]", _tsvQuoter.quoteValue(textFieldName));
+        List<List<String>> bulkData = List.of(
```

Comment:
Do we expect to need to convert many paste-bound texts in this way?  I expect there are a lot of tests out there using plain formatted TSVs

Reply:
There are definitely tons of TSV Strings getting created across our tests. I don't think we need to proactively update things that aren't a problem but we should use these helpers going forward.
That being said, we can't use them universally for editable grid pasting.